cmd/ingress-shim/controller/sync_test.go

```diff
@@ -380,7 +381,7 @@ func TestBuildCertificates(t *testing.T) {
 					DefaultIssuerKind: test.DefaultIssuerKind,
 				},
 			}
-			crts, err := c.buildCertificates(test.Ingress)
+			crts, _, err := c.buildCertificates(test.Ingress)
```

Comment:
Can we update the tests to include a couple of test cases for updating certs? That'll require changing the 'Expected' field on the test struct to be 'ExpectedCreate' and 'ExpectedUpdate'

Reply:
I've added an update test case